extensions/amp-mustache/amp-mustache.md

```diff
@@ -54,7 +54,7 @@ Allows rendering of <a href="https://github.com/janl/mustache.js/">Mustache.js</
 
 ## Syntax
 
-Mustache is a logic-less template syntax. See [Mustache.js docs](https://github.com/janl/mustache.js/) for more details. Some of the core Mustache tags are:
+Mustache is a logic-less template syntax. See [Mustache specification](http://mustache.github.io/mustache.5.html) for more details. Some of the core Mustache tags are:
```

Comment:
Which specification are we following then? Sounds like from the meeting notes it's different based on server / client.

Reply:
The implementation differs based on format we are in AMP4Websites uses JS, AMP4Email depends on Email client. The implementation deviate from spec based on unspecified behaviour

Which is why this only links to the higher level spec instead of the JS implementation.

Reply:
@kristoferbaxter are you ok for me to merge?

Reply:
Absolutely!

Separately we should aim to have a single implementation or test suite to guarantee equivalent execution of templates.